scripts/bash/run_enceladus.sh

```diff
@@ -308,18 +307,27 @@ if [[ -z "$DRY_RUN" ]]; then
         sleep 10
       fi
     fi
-
-    # Log the log location
+    # Log the log locationd
     echo "$CMD_LINE" >> "$TMP_PATH_NAME"
     echo "The log will be saved to $TMP_PATH_NAME"
-
-    # Run the job
-    bash -c "$CMD_LINE 2>&1 | tee -a $TMP_PATH_NAME"
-
-    # Report the log location
-    echo
-    echo "Job has finished. The logs are saved to $TMP_PATH_NAME"
+    # Run the job and return exit status of the last failed command in the subshell pipeline (Issue #893)
+    bash -c "set -o pipefail; $CMD_LINE 2>&1 | tee -a $TMP_PATH_NAME"
+	# Save the exit status of spark submit subshell run
+	EXIT_STATUS="$?"
```

Comment:
Question: Why the `EXIT_STATUS` have to be assigned this indirect way? Cannot it be done directly from the `bash` call?

Reply:
When assigned directly in bash call the exit status returns empty value. Will test further to make sure I don't miss the more elegant solution you suggest

Reply:
We can skip it if we use `$?` directly in the `if` statement. Then again, if you add a print or a call in between you broke the code. I like this.

Reply:
If you tried and didn't work, keep this solution. No need to spend more time on it IMO.
Maybe add a comment explaining, to avoid future enquirers.